### PR TITLE
fix: update urls to new version of project

### PR DESCRIPTION
**Pull request by gikf**
- Updates urls in boilerplate to new routes, per https://github.com/freeCodeCamp/freeCodeCamp/pull/41786.
- Removes part of the user's exercise log url description, that's no longer used.
- Adds script for changing form action url, to one including `uid`, on form submit, as it is in demo project. 
- Code formatting got clean up as well.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

### CODE REVIEW

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks for taking care of this last boilerplate, @gikf. Also, thank you for catching that part of the description that's no longer used (`<strong>{ }</strong> = required,`). I totally missed that the first time around.

Everything LGTM :+1:

When this gets merged, we should also update [this file](https://github.com/freeCodeCamp/demo-projects/blob/master/exercise-tracker/views/index.html) in the demo project repo to match.

File: views/index.html [hunk 1]
```diff
@@ -1,35 +1,46 @@
 <!DOCTYPE html>
 <html>
+  <head>
+    <title>Exercise Tracker | Free Code Camp</title>
```
Comment (scissorsneedfoodtoo) [inline views/index.html:4]: Sorry, noticed one minor thing:

```suggestion
    <title>Exercise Tracker | freeCodeCamp</title>
```

Not a huge deal, though.

**Comment by guspan-tanadi** [comment]
GET user's exercise log: GET /api/users/:_id/logs?[&from][&to][&limit]

probably should become /api/users/:_id/logs?[from][&to][&limit]
As provided in sample project https://exercise-tracker.freecodecamp.rocks

**Comment by scissorsneedfoodtoo** [comment]
Thanks for catching that @guspan-tanadi.

We'll fix that ASAP.

**Pull request merged**